script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -411,7 +411,7 @@ Also, Tohno-kun, you better not be charmed by her!"
 [sha:952bab861a560285d88891b48bb9da932951ae04]{
 -- Page 63, Offset 14956.
 -- 「はい！　いえ、してない、してません！」
-"I am!――not! Not at all!"
+"I am――not! Not at all!"
```

Comment:
Revert. He first says 'I am' and then corrects himself in a hurry.

Reply:
![image](https://user-images.githubusercontent.com/90473809/189362540-17fd92f8-9e36-44e2-8e59-7ebd939ea861.png)
yeah i know but that dash immediately after the exclamation doesnt look too good
maybe "I am! ――No, I'm not! Not at all!" is better?

Reply:
I think it looks better yeah, though I'd change it to an ellipses tbh

Reply:
sure no problem